### PR TITLE
Rt globals ng patches

### PR DESCRIPTION
**Pull request by wingo**
Hey check these three patches and lmk what you think!!!

### CODE REVIEW

**Comment by pmatos** [comment]
Sorry - will only just take a look at these.

File: llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.h [hunk 1]
```diff
@@ -47,25 +47,30 @@ class WebAssemblyTargetLowering final : public TargetLowering {
 
   enum WasmAddressSpace {
     DEFAULT = 0,
-    EXTERNREF = 1,
-    EXTERNREF_GLOBAL = 2,
-    FUNCREF = 3,
-    FUNCREF_GLOBAL = 4,
+    EXTERNREF = 10,
```
Comment (pmatos) [inline llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.h:50]: Why did you decide to number them like this as opposed to sequentially?

**Review by pmatos — COMMENTED** [review]
Besides the one comment - i am thinking about merging and pushing this to phab to move this forward.

File: llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.h (hunk 1 above)
Comment (wingo) [reply llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.h:50]: In order to not update the tests.  however let me check, i have a fresh rebased patch---

**Pull request closed without merging**